app/steps/personal_detail.rb

```diff
@@ -1,6 +1,6 @@
 # frozen_string_literal: true
 
-class IntroductionIntroduceYourself < Step
+class PersonalDetail < Step
```

Comment:
Naming: `PersonalInformation`?  I kind of wish these were just attributes on a `User`. I'm a 1/10 on this, though.